manifests/f/FACEITLTD/FACEITClient/1.34.0/FACEITLTD.FACEITClient.installer.yaml

```diff
@@ -0,0 +1,23 @@
+# Automatically updated by the winget bot at 2024/Mar/02
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: FACEITLTD.FACEITClient
+PackageVersion: 1.34.0
+InstallerLocale: en-US
+Platform:
+- Windows.Desktop
+MinimumOSVersion: 10.0.0.0
+InstallerType: exe
+Scope: machine
+InstallModes:
+- silent
+InstallerSwitches:
+  Silent: --silent
+  SilentWithProgress: --silent
+UpgradeBehavior: install
+Installers:
+- Architecture: x64
+  InstallerUrl: https://faceit-client.faceit-cdn.net/release/FACEIT-setup-latest.exe
+  InstallerSha256: 99495F37784A0428F8301FF21FA4D3675DB06A85081B8AE75D49D08D216F26FF

```

Comment:
```suggestion
  InstallerSha256: 2A24AC8DAF67AE0F8A4F8B1DA6C7B84C47B0DEE06F167704BB7BB5E428571060
```

Reply:
Abandoned my review/suggestion, the HASH you've provided should be good to go... _unless this URL returns a different installer based on the requester's region_... which will be difficult to overcome this issue if that's the case (saying this because the HASH doesn't match the one you've committed).

Lets just hope it works 🥲, and as always, great job @stephengillie